src/components/pages/HomePage.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchDataForAllCountries } from '../../Redux/state/ContinentState';
+import { fetchDataForCountry } from '../../Redux/state/CountryState';
+import '../styles/HomePage.css';
+import showcase from '../images/cov19.png';
+import AllMyCountries from './AllMyCountries';
+
+const HomePage = () => {
```

Comment:
- [ ] The styling of the list on your Homepage does not follow the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore, kindly ensure that cells of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/142708953-8c9be2ee-0745-4f49-9e24-9d7be96df854.png)



**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)